api/db-cdr/generate-cdr/test-data/create_prep_variant_attribute_tables.sql

```diff
@@ -0,0 +1,59 @@
+CREATE OR REPLACE TABLE `aou-res-curation-output-prod.C2022Q4R9.prep_variant_attribute`
+    CLUSTER BY vid
```

Comment:
C2022Q4R9 -> is hard-coded. Is this the intent? or should we have a $BQ_DATASET type variable substitution?

Reply:
Yes, this is a one time creation of synth data for test environment. It's not intended to be dynamic and notice that its in the test-data folder. Just want to keep the scripts in case we have to so this again.

Reply:
got it!